### ISSUE TITLE
Did you forget a `,` here?

### ISSUE THREAD

**Issue by dchambers** [reasonml-community/graphql-ppx#226]
Hi, I'm just trying to find my way with ReScript and am getting this error after installing `graphql-ppx` as per the instructions:

![image](https://user-images.githubusercontent.com/283294/103330660-6bbd1d00-4a5a-11eb-9a65-0a0d3c7d4785.png)

From the terminal:

```res
>>>> Start compiling
bsb: [1/3] src/index.ast
FAILED: src/index.ast

  Syntax error!

  19 │ })
  20 │
  21 │ [%graphql {|
  22 │   query UserQuery {
  23 │     user {

  Did you forget a `,` here?

FAILED: cannot make progress due to previous errors.
>>>> Finish compiling(exit: 1)
```

My `bsconfig.json` looks like this:

```json
{
  "sources": {
    "dir" : "src",
    "subdirs" : true
  },
  "package-specs": {
    "module": "es6"
  },
  "suffix": ".bs.js",
  "ppx-flags": ["@reasonml-community/graphql-ppx/ppx"],
  "bs-dependencies": [
    "@reasonml-community/graphql-ppx",
    "bs-fetch",
    "bs-webapi"
  ],
  "generate-merlin": false,
  "graphql": {
    "schema": "../../libs/graphql-schema/schema.json"
  },
  "warnings": {
    "error" : "+101"
  }
}
```

Any idea what I might be doing wrong here?

**Comment by jfrolich** [comment]
If you are using `res` instead of `re` syntax, you have to use it like this 
```rescript
%graphql(`
...
`)
```

**Comment by dchambers** [comment]
> If you are using `res` instead of `re` syntax, you have to use it like this
> 
> ```rescript
> %graphql(`
> ...
> `)
> ```

That worked! Thank you kindly 🙏.

BTW, I'm not sure I did the right thing, but in my case the `cast` function wasn't available (`The value cast can't be found in <module>`) so I ended up using `unsafe_fromJson`, which sounded like it might be the right thing and made the compiler happy.

**Comment by jfrolich** [comment]
> > If you are using `res` instead of `re` syntax, you have to use it like this
> > ```rescript
> > %graphql(`
> > ...
> > `)
> > ```
> 
> That worked! Thank you kindly 🙏.
> 
> BTW, I'm not sure I did the right thing, but in my case the `cast` function wasn't available (`The value cast can't be found in <module>`) so I ended up using `unsafe_fromJson`, which sounded like it might be the right thing and made the compiler happy.

Yes. Where did you find `cast` documented? (It actually never existed, but I used in some documentation that I wrote before writing the implementation :))

**Comment by dchambers** [comment]
> Yes. Where did you find `cast` documented? (It actually never existed, but I used in some documentation that I wrote before writing the implementation :))

It's on the [Getting Started](https://graphql-ppx.com/docs/getting-started) page as you see here:

![image](https://user-images.githubusercontent.com/283294/103392800-75588a80-4b17-11eb-982d-41583dbcd0c7.png)

Thanks again for an amazing tool! 🚀

**Comment by jfrolich** [comment]
Thanks, fixed in d060d5728d67882910529591e21258bb25b430ff!
